lib/routes/estados.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
'use strict' is unnecessary inside of modules.

[Read more about it here.](http://eslint.org/docs/rules/strict)